### PR TITLE
LPS-78950 Remove usage of gender info across the code - WIP

### PR DESCRIPTION
**Pull request by JorgeFerrer**
Partial implementation.

Needs completion, specially reviewing all the usages across the code of the "male" field.

### CODE REVIEW

**Comment by JorgeFerrer** [comment]
ci:test-relevant

File: modules/apps/opensocial/opensocial-portlet/docroot/WEB-INF/src/com/liferay/opensocial/shindig/service/LiferayPersonService.java [hunk 1]
```diff
@@ -281,9 +281,7 @@ protected Person getUserPerson(
 
 		sb.append(securityToken.getDomain());
 		sb.append(PortalUtil.getPathImage());
-		sb.append("/user_");
-		sb.append(user.isFemale() ? "female" : "male");
-		sb.append("_portrait?img_id=");
+		sb.append("/user_male_portrait?img_id=");
```
Comment (mdelapenya) [inline modules/apps/opensocial/opensocial-portlet/docroot/WEB-INF/src/com/liferay/opensocial/shindig/service/LiferayPersonService.java]: Could  this be renamed to `user_portrait`?

Comment (JorgeFerrer) [reply modules/apps/opensocial/opensocial-portlet/docroot/WEB-INF/src/com/liferay/opensocial/shindig/service/LiferayPersonService.java]: Fully agreed. 

This part is unfinished and @drewbrokke will continue working on it.

**Comment by drewbrokke** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by drewbrokke** [comment]
First PR sent here: https://github.com/brianchandotcom/liferay-portal/pull/66186.  Only deprecates methods, does not remove usages.

File: modules/apps/opensocial/opensocial-portlet/docroot/WEB-INF/src/com/liferay/opensocial/shindig/service/LiferayPersonService.java (hunk 1 above)
Comment (drewbrokke) [reply modules/apps/opensocial/opensocial-portlet/docroot/WEB-INF/src/com/liferay/opensocial/shindig/service/LiferayPersonService.java]: @mdelapenya that's the plan. See https://github.com/brianchandotcom/liferay-portal/pull/66186/commits/e22aebaf16ff25446d4858ab6635d32a572e673f

**Pull request closed without merging**